mdata/cache/accnt/if.go

```diff
@@ -20,3 +20,8 @@ type EvictTarget struct {
 	Metric string
 	Ts     uint32
 }
+
+func NewAccnt(maxSize uint64) Accnt {
+	// currently we only know NewFlatAccnt
+	return NewFlatAccnt(maxSize)
+}
```

Comment:
The idea is that it should be simple to add an alternative accounting mechanism, that's why it's behind an interface. That's only called once at initialization so I'm not worried about the additional stack frame.